dali/operators/reader/video_reader_decoder_op_test.cc

```diff
@@ -166,6 +166,75 @@ TEST_F(VideoReaderDecoderCpuTest, CpuVariableFrameRate_CpuOnlyTests) {
   }
 }
 
+TEST_F(VideoReaderDecoderGpuTest, GpuVariableFrameRate) {
+  const int batch_size = 4;
```

Comment:
It seems to have a lot in common with `CpuVariableFrameRate_CpuOnlyTests`. Maybe something can be extracted to a common function.

Reply:
Version you commented on was just a copy of the CPU test. When I fully adjusted it to work properly with GPU operator, extracting it would mean either making a lot of templated mess or some wierd flexing with extracting small common pieces. 
For now, I'd like to leave it as it as, since it makes it easier to debug. When we add more tests (more video formats, more output types etc.) I'll do something to reduce code duplication in these test.

Reply:
👍